app/src/App.js

```diff
@@ -3,36 +3,52 @@ import "./App.css";
 import * as React from "react";
 
 import Account from "./Account.js";
+import gcal from "./api/ApiCalendar";
 import Planner from "./components/Planner.js";
 // import * as apiClient from "./apiClient";
 
 const App = () => {
-  const [isSignedIn, setIsSignedIn] = React.useState(false);
-  const [user, setUser] = React.useState({});
-
-  function logOut() {
-    setUser({});
-    setIsSignedIn(false);
-    console.log("logout successful");
-  }
+  const [isAuthenticated, setIsAuthenticated] = React.useState(gcal.sign);
 
   React.useEffect(() => {
-    console.log(user);
-  }, [user, isSignedIn]);
+    gcal.onLoad(() => {
+      setIsAuthenticated(gcal.gapi.auth2.getAuthInstance().isSignedIn.get());
+      gcal.listenSign((sign) => setIsAuthenticated(sign));
+    });
+  }, []);
 
   return (
     <main className="App">
       <h1>Here it is!</h1>
-      <Account
-        user={user}
-        setUser={(e) => setUser(e)}
-        logOut={() => logOut()}
-        isSignedIn={isSignedIn}
-        setIsSignedIn={(e) => setIsSignedIn(e)}
-      />
-      {isSignedIn ? <Planner /> : <p>Yo, sign in already.</p>}
+      <Login {...{ isAuthenticated }} />
+      {isAuthenticated ? <Events /> : null}
     </main>
   );
 };
 
+const Login = ({ isAuthenticated }) =>
+  isAuthenticated ? (
+    <button onClick={gcal.handleSignoutClick}>Log out</button>
+  ) : (
+    <button onClick={gcal.handleAuthClick}>Log in</button>
+  );
+
+const Events = () => {
+  const [events, setEvents] = React.useState([]);
+
+  React.useEffect(() => {
+    gcal
+      .listUpcomingEvents(10)
+      .then(({ result: { items } }) => setEvents(items));
+  }, []);
+
+  return events.length === 0 ? null : (
+    <ul>
+      {events.map((event) => (
+        <li key={event.id}>{event.summary}</li>
+      ))}
+    </ul>
+  );
```

Comment:
You might not need the ternary here, since if `events.length` is zero then `events.map` will iterate 0 times, which (I think) should have the same effect as returning `null`.